CHANGELOG-1.1.md

```diff
@@ -1,10 +1,39 @@
+# TiDB Operator v1.1.0-beta.2 Release Notes
+
+This is a pre-release of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Changes since v1.1.0-beta.1
+
+## Action Required
+
+- `--default-storage-class-name` and `--default-backup-storage-class-name `are abandoned, storage class defaults to Kubernetes default storage class right now. If you have set default storage class different than Kubernetes default storage class, please set them explicitly in your tidb cluster helm or YAML files. ([#1581](https://github.com/pingcap/tidb-operator/pull/1581), [@cofyc](https://github.com/cofyc))
+
+## Other Notable Changes
+
+- Allow users to configure affinity and tolerations for `Backup` and `Restore` ([#1737](https://github.com/pingcap/tidb-operator/pull/1737), [@Smana](https://github.com/Smana))
+- Make AdvancedStatefulSet and Admission Webhook could work together. ([#1640](https://github.com/pingcap/tidb-operator/pull/1640), [@Yisaer](https://github.com/Yisaer))
+- Add a basic deployment example of TiDB cluster ([#1573](https://github.com/pingcap/tidb-operator/pull/1573), [@aylei](https://github.com/aylei))
+- Support TidbCluster Auto-scaling feature based by CPU average utilization load. ([#1731](https://github.com/pingcap/tidb-operator/pull/1731), [@Yisaer](https://github.com/Yisaer))
+- Support user-defined tidb server/client certificate ([#1714](https://github.com/pingcap/tidb-operator/pull/1714), [@weekface](https://github.com/weekface))
+- Add an option for tidb-backup chart to allow reusing existing PVC or not for restore ([#1712](https://github.com/pingcap/tidb-operator/pull/1712), [@sre-bot](https://github.com/sre-bot))
```

Comment:
I think we should refine the log by changing the `sre-bot` author.

Reply:
ah, I'll fix this.

Reply:
Yes, with the generated release note, we still should refine some words and sentences. The PR and author should be the original PR and author, other than sre-bot.

Reply:
I'll figure out how to filter out `sre-bot`.

Reply:
fixed